BREAKING-CHANGES.md

```diff
@@ -72,6 +72,33 @@ This is a quick summary of the sections below:
   - MSRV is now 1.63.0
   - `List` no longer ignores empty strings
 
+## Unreleased
+
+### `Bar::label()` and `BarGroup::label()` now accepts `Into<Line<'a>>`. `Bar::text_value` now accepts `Into<String>` ([#1471])
```

Comment:
Maybe we can split this to two entries in the document (for `Into<Line>` and `Into<String>`)?

Reply:
I thought the same, but I wasn't sure if mentioning the same PR in two entries would be okay. I'll go ahead and do it.